/drone:
  joint_state_controller:
    type: "joint_state_controller/JointStateController"
    publish_rate: 50


  drone_mech_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - /drone/link1_to_link2
      - /drone/link2_to_link3
      - /drone/link3_to_link4
      - /drone/link4_to_drone
    constraints:
        goal_time: &goal_time 0.6
        stopped_velocity_tolerance: &stop_vel_tol 0.05
        link1_to_link2: {trajectory: &traj_tol 0.05, goal: &goal_tol 0.05}
        link2_to_link3: {trajectory: *traj_tol, goal: *goal_tol}
        link3_to_link4: {trajectory: *traj_tol, goal: *goal_tol}
        link4_to_itself: {trajectory: *traj_tol, goal: *goal_tol}
    stop_trajectory_duration: &stop_traj_dur 0.2
    state_publish_rate:  50
    action_monitor_rate: 10